msyncd/synchronizer.cpp

```diff
@@ -115,8 +115,9 @@ bool Synchronizer::initialize()
             this, SLOT(slotSyncStatus(QString, int, QString, int)),
             Qt::QueuedConnection);
 
+    // use queued connection because the profile will be stored after the signal
     connect(&iProfileManager ,SIGNAL(signalProfileChanged(QString,int,QString)),
-            this, SIGNAL(signalProfileChanged(QString,int,QString)));
+            this, SLOT(slotProfileChanged(QString,int,QString)), Qt::QueuedConnection);
```

Comment:
you've changed this to the slot instead of the signal - I assume this was just a bug in the original code, and not a trampoline for the Out Of Process plugins?